core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/instance/InstanceGroup.java

```diff
@@ -59,6 +60,9 @@
     @Enumerated(EnumType.STRING)
     private InstanceGroupType instanceGroupType = InstanceGroupType.CORE;
 
+    @Enumerated(EnumType.STRING)
```

Comment:
please use converter as it's more safe for future changes. see: https://github.com/hortonworks/cloudbreak/blob/master/common/src/main/java/com/sequenceiq/cloudbreak/converter/TunnelConverter.java

Reply:
Thanks, done!